src/models/layerMetadata/layerMetadata.ts

```diff
@@ -1,5 +1,7 @@
+import { keys } from 'ts-transformer-keys';
```

Comment:
rename to layerRasterMetadata as the convention above suggests 
![image](https://github.com/user-attachments/assets/4b2b2606-2eb7-43cd-8e7b-9f5a24e1d099)